modules/managed-cluster-notification-view-in-hcc.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa-cluster-notifications.adoc
+// * osd_cluster_admin/osd-cluster-notifications.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="managed-cluster-notification-view-hcc_{context}"]
+= Viewing cluster notifications using the {hybrid-console}
+
+Cluster notifications provide important information about the health of your cluster. You can view notifications that have been sent to your cluster in the **Cluster history** tab on the {hybrid-console}.
+
+.Prerequisites
+* You are logged in to the {hybrid-console-second}.
+
+.Procedure
+. Navigate to the link:https://console.redhat.com/openshift[Clusters] page of the {hybrid-console-second}.
+. Click the name of your cluster to go to the cluster details page.
+. Click the **Cluster history** tab.
++
+Cluster notifications appear under the Cluster history heading.
+. Optional: Filter for relevant cluster notifications
++
+Use the filter controls to hide cluster notifications that are not relevant to you, so that you can focus on your area of expertise or on resolving a critical issue. You can filter by text in the notification description, severity level, notification type, or which system or person triggered the notification.
```

Comment:
@laubai Hi, There is also a date filter in cluster history section and useful for the notification filter. I could see we haven't specified any details about this. I think it would be good to mention the same here as well . WDYT?
![image](https://github.com/openshift/openshift-docs/assets/93653094/30603bc6-1e55-46d7-92dd-c2ad6870f8a1)

Reply:
Cheers, added "when the notification was received" to this list.